experimental/PropertyDDS/examples/partial-checkout/webpack.config.cjs

```diff
@@ -22,6 +22,12 @@ module.exports = (env) => {
 					test: /\.tsx?$/,
 					loader: "ts-loader",
 				},
+				{
+					test: /\.m?js/,
+					resolve: {
```

Comment:
What is this for?

Reply:
When adding '"type": "module"', WebPack starts enforcing that imports specify a file extension.  Unfortunately, it does this for all import statements in transitive dependencies and starts running into trouble with packages that haven't been fully converted to ESM.

I'm not sure if this is by design, or if WebPack is being confused by PNPM workspace symlinks and is checking dependencies as-if they were part of the root package.

I added a comment to this and other pre-existing usages in #19582.